client/src/test/java/com/alibaba/nacos/client/StringUtilsTest.java

```diff
@@ -0,0 +1,62 @@
+package com.alibaba.nacos.client;
+
+import com.alibaba.nacos.client.utils.StringUtils;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import static org.junit.Assert.*;
+import static com.alibaba.nacos.client.utils.StringUtils.*;
+
+public class StringUtilsTest {
+
+    @Test
+    public void testisNotBlank() {
+        assertTrue(isNotBlank("foo"));
+
+        assertFalse(isNotBlank(" "));
+        assertFalse(isNotBlank(null));
```

Comment:
What about `"", "\t", "\n"`?

Reply:
I didn't put them in as there was 100% coverage, but I can put them in if you like :)